ngx-style.pl

```diff
@@ -144,7 +144,7 @@
             }
 
             # 3.
-            if ($line =~ /\(\w+\)\w+/) {
+            if ($line =~ /\([a-zA-Z0-9_*]+( \*)?\)[a-zA-Z_]+/) {
```

Comment:
This looks wrong. We should give warning/error messages for things like `(foo*)` or `(foo    *)`.

Reply:
this is check in other place. we check （type)p  or (type *)p here.